src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/sessions/CreateSessionTemplateDto.kt

```diff
@@ -51,9 +51,6 @@ data class CreateSessionTemplateDto(
   @Schema(description = "list of group references for permitted session location groups", required = false)
   val locationGroupReferences: List<String>? = listOf(),
 
-  @Schema(description = "enhanced privilege", example = "true", required = true)
-  val enhanced: Boolean,
-
```

Comment:
dont we need this?

Reply:
Updated to include incentiveLevelGroupReferences.